<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="TableView" name="tableView">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="graphicsView_3">
          <property name="backgroundBrush">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>232</red>
             <green>231</green>
             <blue>231</blue>
            </color>
           </brush>
          </property>
          <property name="renderHints">
           <set>QPainter::NonCosmeticDefaultPen</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="listView">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="8" column="4">
           <widget class="QCheckBox" name="openGLCompatible">
            <property name="text">
             <string>OpenGLCompatible</string>
            </property>
           </widget>
          </item>
          <item row="8" column="3">
           <widget class="QCheckBox" name="preferOutline">
            <property name="text">
             <string>PreferOutline</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QRadioButton" name="forceIntegerMetrics">
            <property name="text">
             <string>ForceIntegerMetrics</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QRadioButton" name="preferQuality">
            <property name="text">
             <string>PreferQuality</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Font</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QCheckBox" name="preferDefault">
            <property name="text">
             <string>PreferDefault</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QCheckBox" name="preferBitmap">
            <property name="text">
             <string>PreferBitmap</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QRadioButton" name="preferMatch">
            <property name="text">
             <string>PreferMatch</string>
            </property>
            <property name="autoExclusive">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="4">
           <widget class="QCheckBox" name="noSubpixelAntialias">
            <property name="text">
             <string>NoSubpixelAntialias</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QCheckBox" name="noAntialias">
            <property name="text">
             <string>NoAntialias</string>
            </property>
           </widget>
          </item>
          <item row="9" column="4">
           <widget class="QCheckBox" name="noFontMerging">
            <property name="text">
             <string>NoFontMerging</string>
            </property>
           </widget>
          </item>
          <item row="6" column="4">
           <widget class="QCheckBox" name="preferAntialias">
            <property name="text">
             <string>PreferAntialias</string>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QCheckBox" name="preferDevice">
            <property name="text">
             <string>PreferDevice</string>
            </property>
           </widget>
          </item>
          <item row="9" column="3">
           <widget class="QCheckBox" name="forceOutline">
            <property name="text">
             <string>ForceOutline</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="5">
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="plainText">
             <string>Широкая электрификация южных губерний даст мощный толчок подъёму сельского хозяйства.
Jackdaws love my big sphinx of quartz.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
